domain/src/main/java/org/oppia/domain/exploration/ExplorationRetriever.kt

```diff
@@ -14,10 +14,10 @@ import org.oppia.domain.util.StateRetriever
 import java.io.IOException
 import javax.inject.Inject
 
-const val TEST_EXPLORATION_ID_5 = "DIWZiVgs0km-"
-const val TEST_EXPLORATION_ID_6 = "test_exp_id_6"
-const val TEST_EXPLORATION_ID_30 = "30"
-const val TEST_EXPLORATION_ID_7 = "test_exp_id_7"
+const val TEST_EXPLORATION_ID_5 = "0"
+const val TEST_EXPLORATION_ID_6 = "1"
+const val TEST_EXPLORATION_ID_30 = "2"
```

Comment:
Can we make the names of the variables same as the values? i.e. change TEST_EXPLORATION_ID_30 to TEST_EXPLORATION_ID_2?

Reply:
I think we should, but not in this PR. #370 is the first step toward this, and after that ID management should be much better.

Reply:
I think we should, but not in this PR. #370 is the first step toward this, and after that ID management should be much better.